Newsletters: add 60 (2019-08-21)

harding opened the pull request:
- [x] Finish compatibility matrix stuff
- [x] Check for C-Lightning release announcement

moneyball commented:
initial review LGTM

jnewbery commented on _posts/en/newsletters/2019-08-21-newsletter.md:
```diff
@@ -0,0 +1,107 @@
+
+- [Bitcoin Core #15986][] extends the `getdescriptorinfo` RPC with an
+  additional `checksum` field that returns the checksum for the
+  descriptor provided by the user.  This differs from the checksum
```
I don't think "This differs from the checksum described in Newsletter #34" is the clearest way to introduce this. Suggested alternative:

"Bitcoin Core #15986 extends the getdescriptorinfo RPC described in Newsletter #34. getdescriptorinfo returns the _normalized_ descriptor (which removes private keys and makes other modifications) along with its checksum. It now also returns a `checksum` field which gives the checksum of the _unnormalized_ descriptor provided by the user. Descriptor checksums follow a # character at the end of the string as described in the output script descriptor documentation."

jnewbery commented on _posts/en/newsletters/2019-08-21-newsletter.md:
```diff
@@ -0,0 +1,107 @@
+The *News* section from this week's newsletter introduced a new feature
+on the Optech website.  {% include specials/bech32/23-compat.md %}
+
+## Notable code and documentation changes
```
nit: add Bitcoin Core n16060 with one sentence referring to the news section.

jnewbery commented:
ACK ef2778028e47385e3090375f9dc12bef9d27312f. Great stuff!

practicalswift commented:
ACK ef2778028e47385e3090375f9dc12bef9d27312f

Thanks for yet another nice newsletter!

adamjonas commented on _posts/en/newsletters/2019-08-21-newsletter.md:
```diff
@@ -0,0 +1,109 @@
+  of all active Bitcoin miners (at the time of writing), so this is
+  considered to be both very unlikely and indicative of a threat that
+  could prevent consensus formation anyway.  The hardcoding, which is
+  similar to the [BIP90][] hardcoding made a couple years ago,
```
a couple years -> a couple of years

adamjonas commented:
ACK ef27780. Looks good!

practicalswift commented:
ACK 44f6790c9cc55c67268d983e21b1adfc1de521af

harding commented:
Rebased on master, addressed FIXMEs, and added an insight about bech32 support from the compat table now that it's all in one place (that 74% of the tools we surveyed support sending to a bech32 address).

jnewbery commented on _posts/en/newsletters/2019-08-21-newsletter.md:
```diff
@@ -0,0 +1,107 @@
+- [LND #3391][] always returns the same error message for failed
+  payments in order to avoid leaking whether or not an invoice exists.
+  See [BOLTs #516][] for more details about the
+  information leak and the [BOLTs #608][] for a related leak.
```
```suggestion
  information leak and [BOLTs #608][] for a related leak.
```

harding commented:
Added release announcement for C-Lightning 0.7.2.1.

jnewbery commented:
ACK 6f9b93695ea557a6bc791daf8d9c2fc176bf03ea

bitschmidty commented:
tACK 6f9b93695ea557a6bc791daf8d9c2fc176bf03ea

jnewbery commented:
A newsletter merge without a rhyme?
@bitschmidty, surely you know that's a crime!

@moneyball, @harding and @adamjonas on the edge of their seats,
Waiting for your Wednesday verbal treats.

bitschmidty commented:
@jnewbery Looks like you have taken the torch!

Optech announced the compatibility matrix
Some wallets gotta fix the basics

Soft fork blocks are now hardcoded
So consensus code doesnt get bloated

@harding great newsletter, @adamjonas and @jnewbery thanks for the reviews!
